packages/design-system/src/components/slider/index.js

```diff
@@ -248,13 +252,15 @@ function Slider({
   ]);
 
   const printValue = `${value}${suffix}`;
-
   return (
     <Tooltip
       title={printValue}
       placement={PLACEMENT.TOP}
+      popupZIndexOverride={11}
```

Comment:
This is currently broken in main. Used in the grid view, which is a Modal. Should I add an enum to constants/zIndex. Probably.